README.md

```diff
@@ -1,8 +1,13 @@
 # Bloxone Azure Templates
 
 ## Getting Started
+ 
+To prepare ARM templates for QA team use patch-for-qa.sh script from utils directory:
+```
+$ utils/patch-for-qa.sh accessPortal="" configServer="" notificationServer=""
+```
 
 To create zip archive use archive.sh script from utils directory:
 ```
-$ utils/archive.sh accessPortal="" configServer="" imageOffer="" imagePublisher="" imageSku="" imageVersion="" notificationServer="" pid=""
+$ utils/archive.sh imageOffer="" imagePublisher="" imageSku="" imageVersion="" pid=""
```

Comment:
add more description about where will we get these value, especially `pid` it is not self explanatory

Reply:
Added additional information.